compat/src/render.js

```diff
@@ -11,6 +11,13 @@ export const REACT_ELEMENT_TYPE =
 	0xeac7;
 
 const CAMEL_PROPS = /^(?:accent|alignment|arabic|baseline|cap|clip(?!PathU)|color|dominant|fill|flood|font|glyph(?!R)|horiz|image|letter|lighting|marker(?!H|W|U)|overline|paint|pointer|shape|stop|strikethrough|stroke|text(?!L)|transform|underline|unicode|units|v|vector|vert|word|writing|x(?!C))[A-Z]/;
+const DOUBLE_CLICK = /ondoubleclick/;
+const CHANGE_INPUT = /^onchange(textarea|input)/;
+const ON_ANI = /^on(Ani|Tra|Tou|BeforeInp|Compo)/;
+const ON_BLUR = /^onblur$/;
+const ON_FOCUS = /^onfocus$/;
+const ON_INPUT = /^oninput$/;
```

Comment:
Since we're lower-casing preemptively now, I bet we could get away with inlining these as straight equality checks (`lowerCased === 'onfocus'`). Should be faster than the regex.

Reply:
Good shout, the camel props one is still the one hurting perf the most 😅